include/prereq-build.mk

```diff
@@ -183,8 +183,8 @@ $(eval $(call FindHostCommand,*readlink,Please install GNU 'readlink', \
 	readlink --version | grep GNU, \
 	greadlink --version | grep GNU))
 
-$(eval $(call SetupHostCommand,perl,Please install Perl 5.x, \
-	perl --version | grep "perl.*v5"))
+$(eval $(call SetupHostCommand,perl,Please install Perl >= 5.20, \
+	perl --version | grep 'perl.*v5.[2-9][0-9]'))
```

Comment:
why?

Reply:
i feel we should require a version of perl thats not *ancient* due to security bugs and syntax differences over time, a good example out of many:

https://www.effectiveperlprogramming.com/2014/07/perl-5-20-introduces-keyvalue-slices/